packages/cli/src/swc/__tests__/options.test.ts

```diff
@@ -1,5 +1,8 @@
 import type { Options } from "@swc/core";
 import deepmerge from "deepmerge";
+import fs from "fs";
+
+jest.mock("fs");
```

Comment:
This seems wrong.
https://github.com/swc-project/swc/issues/5205

Reply:
I'm not sure I get the problem here. Can you please elaborate?